### DISCUSSION TITLE
Only Certain Pages Showing

### DISCUSSION THREAD

**Discussion by ariyin** [community/community#69079] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello,

I've tried various methods to fix this problem, but none seem to work. Some of the pages seem to work (Home, About, Projects). However, specific project pages don't work. If I click on a project, it tells me that the page doesn't exist. And if I press the back button on a 404 page, pages that seem to work also turn into 404 errors. Typing in a link of any page except for the Home page will also give a 404 error, but navigating to those pages (About, Projects) through the site itself seems to work.

This seems really wacky and I'm not sure how to fix it.

Links:
Site: https://ariyin.github.io/
Repo: https://github.com/ariyin/ariyin.github.io

Any help would be appreciated. Thank you!

**Comment by vishalkrishna8** [comment]
Hey @ariyin, you can slove this issue by taking some steps:
Star with verifying that the file names and paths are accurate, paying attention to case sensitivity. Review your website's links and navigation menus to ensure they correctly point to the intended URLs. Inspect your GitHub repository to confirm that all required files are present. Additionally, clearing your browser's cache and cookies may help resolve the issue. Double-check your GitHub Pages configuration in your repository settings and your Jekyll configuration if applicable. If you have a custom 404 error page, ensure it's set up correctly. Testing on different web browsers can also provide insights. Lastly, check the status of your GitHub Pages build. 
Hope this helps you!

**Reply by ariyin** [reply]
1. All of the navigation works on localhost, so I'm wondering if Github Pages requires something else -> this is the fault I'm trying to pursue
2. I've cleared my browser's cache and cookies and nothing happened 
3. The GitHub Pages configuration is set to gh-pages root, which I believe is the correct one
4. I have no Jekyll configuration
5. I have no custom 404 error page
6. I tried looking at the website on Safari instead of Chrome, but other than a font not working, the pages are also just as unaccessible
7. The status of the build is fine, it passed all checks

**Reply by ariyin** [reply, marked as answer]
I've managed to fix the issue of navigating to specific project pages (had to use Link from React instead of href), but I still have the problem where if I type the link into the search bar, GitHub gives me a 404 error, but navigating to the page from the home page works fine. Not sure how to fix this one.

EDIT: Found the solution to the aforementioned problem with this repo: https://github.com/rafgraph/spa-github-pages. If you do what the instructions say, the problem should be fixed.
